### ISSUE TITLE
About the network

### ISSUE THREAD

**Issue by liuxuexun** [yz93/LAVT-RIS#33]
作者您好，非常感谢您开源了您的优秀工作，我最近在做相关的研究，想请教这个工作的一些问题：

1. 作者您在某个issue提到如果batchsize太小的话网络训不起来，具体是性能下降多少？
2. 如果backbone不采用预训练网络，而是随机初始化，是否也能使网络收敛？
3. 对于损失函数的权重，您是如何决定为[0.9, 1.1]的呢？图片中有的物体过小，只靠权重可以克服吗？

我是做其他领域的相关研究接触到了您的工作，对先前的工作和常用方法不太了解，还请您原谅并赐教！

以下是上方文字的机翻：(The following is a machine translation of the text above)

Hello author, thank you very much for opening up your excellent work. I am currently conducting related research and would like to ask some questions about this work:

1. The author mentioned in a certain issue that if the batch size is too small, the network cannot be trained. Specifically, how much does the performance decrease?

2. If the backbone does not use pre trained networks but is initialized randomly, can the network also converge?

3. How did you determine the weight of the loss function to be [0.9, 1.1]? Some objects in the picture are too small, can they be overcome solely by weight?

I came into contact with your work while conducting related research in other fields. I am not very familiar with my previous work and commonly used methods. Please forgive me and give me your advice!
